ludwig/features/text_feature.py

```diff
@@ -392,7 +381,7 @@ def postprocess_predictions(
         backend,
     ):
         # todo: refactor to reuse SequenceOutputFeature.postprocess_predictions
-        level_idx2str = "{}_{}".format(self.level, "idx2str")
+        level_idx2str = f"{self.level}_idx2str"
```

Comment:
maybe idx2str and str2idx can become constants

Reply:
Sure, happy to follow up on this in a separate PR. `idx2str` and `str2idx` are used in many files.